src/Package/Impl/Feedback/SendFrownCommand.cs

```diff
@@ -23,7 +18,11 @@ internal sealed class SendFrownCommand : SendMailCommand {
 
         protected override void Handle() {
             string zipPath = DiagnosticLogs.Collect();
-            SendMail(_disclaimer, "RTVS Frown", zipPath);
+
+            var generalData = new StringWriter();
+            DiagnosticLogs.WriteGeneralData(generalData, detailed: false);
+
+            SendMail(string.Format(Resources.SendFrownEmailBody, zipPath, generalData.ToString(), zipPath), "RTVS Frown", zipPath);
```

Comment:
SendFrownEmailBody only has 2 replacement tags, you are passing in 3 parameters (zip path twice).